mitodl/unified-ecommerce
# Adds some CI actions back in

### What are the relevant tickets?

n/a

### Description (What does it do?)

When I set up this repo, I copied over the GitHub workflows from another project - but, they would have required some additional work to get them to work right with this project, so they got placed in a folder that GitHub wouldn't pick up and use. This moves the CI part of that out and makes the necessary changes to make them work (now that we have actual code to do stuff with).

This also updates the `README.md` so that it matches reality. 

### How can this be tested?

CI should work but this should also cover the things we generally want covered in a CI pipeline. 

Some notes:
- There's no frontend stuff so there's no testing or linting or anything for JavaScript assets. When that stuff is in place, it can be added back in. (Some of that is in there but commented out.)
- The `ci.yaml` from Open had a separate step for checking that the OpenAPI schema was generated and then makes sure it's formatted/etc. This has been pulled; the standard python-tests step checks for the schema as it is and there's (again) no working frontend build stuff so eslint/etc. was failing. At this stage, having a pretty API client isn't a huge priority; this will be added back in when there's generally frontend stuff again.
